### ISSUE TITLE
Error trying to run without spa

### ISSUE THREAD

**Issue by ndkeen** [E3SM-Project/scream#1733]
With June 10th master, which includes changes to make running without SPA easier, I wanted to try a run without SPA. I added these lines:

```
./atmchange do_aerosol_rad=false
./atmchange do_prescribed_ccn=false
./atmchange do_predict_nc=false
./atmchange mac_aero_mic::atm_procs_list="(shoc,cldFraction,p3)"
```

But I see fail:

```
 17: terminate called after throwing an instance of 'std::logic_error'
 17:   what():  /global/cfs/cdirs/e3sm/ndk/se09-jun10/externals/ekat/src/ekat/util/ekat_factory.hpp:141: FAIL:
 17: it!=m_register.end()
 17: [N4ekat7FactoryIN6scream20AtmosphereDiagnosticENS_21CaseInsensitiveStringESt10shared_ptrIS2_EJRKNS_4CommERKNS_13ParameterListEEEE] Error!
 17:         The key 'aero_g_sw' is not associated to any registered product.
 17:         The list of registered product is: PotentialTemperature
 17:         Did you forget to register it?
 17: 
 17: 
 17: Program received signal SIGABRT: Process abort signal.
```

Which I recall was a var we prev needed to set to 0.0 in yaml input.

**Comment by PeterCaldwell** [comment]
It looks like run/data/scream_output.yaml still contains SPA variables in its default output request. You need to delete them. Hang on a sec while I look at which variables to delete...

**Comment by AaronDonahue** [comment]
This is probably because those variables are still set to be run as output.  In our discussion on Slack today it looks like we may supress those as typical output anyway so the error should go away.  In the meantime if you edit `run/data/scream_output.yaml` and remove the whole section labeled as SPA then you shouldn't see those errors.

**Comment by AaronDonahue** [comment]
> It looks like run/data/scream_output.yaml still contains SPA variables in its default output request. You need to delete them. Hang on a sec while I look at which variables to delete...

I organized the output by process so it should be easier to figure out which to delete.  Just navigate to those labeled as SPA

**Comment by PeterCaldwell** [comment]
What Aaron said... The list of vars to delete can also be seen at:
https://github.com/E3SM-Project/scream/blob/47740c818deb71b0244ed30de3bfcfb0276dbe2b/components/scream/data/scream_default_output.yaml#L27

**Comment by ndkeen** [comment]
So I thought we were trying to make it easier to run without SPA. There is not a way to do this via atmchange?

**Comment by AaronDonahue** [comment]
I think the discussion in issue #1734 will resolve this issue.

**Comment by AaronDonahue** [comment]
But to the point of making it easier to run without SPA, does it make sense for us to come up with a SPA-less compset that will by default set all of the parameters we need without the use of `atmchange`?  It sseems like something we will do often and a Compset is a pretty easy change.

**Comment by PeterCaldwell** [comment]
I really like the idea of creating a no-SPA compset. Brilliant, Aaron!

**Comment by ndkeen** [comment]
I will let others decide if a nospa compset makes sense (seems heavyweight to me, but you said it's easy). I just think it would be nice to have an easy way to turn off SPA. it may be that it is only needed in short term to debug memory issues and once resolved, we won't need it -- but then again, might always be nice to have the option.

**Comment by AaronDonahue** [comment]
Should be addressed by issue #1736
